packages/contracts-bedrock/test/L1/DelayedVetoable.t.sol

```diff
@@ -74,14 +74,19 @@ contract DelayedVetoable_Getters_Test is DelayedVetoable_Init {
 contract DelayedVetoable_Getters_TestFail is DelayedVetoable_Init {
     /// @dev Check that getter calls from unauthorized entities will revert.
     function test_getters_notZeroAddress_reverts() external {
+        // nosemgrep: sol-style-use-abi-encodecall
         vm.expectRevert(abi.encodeWithSelector(Unauthorized.selector, initiator, address(this)));
         delayedVetoable.initiator();
+        // nosemgrep: sol-style-use-abi-encodecall
```

Comment:
Let's add:

```
      - pattern-not: vm.expectRevert(abi.encodeWithSelector(...));
```

to the semgrep rule so that we don't need to do this all over the place, since this is a common pattern